src/transform/CollapseTable.css

```diff
@@ -80,7 +80,7 @@ table.infobox caption {
 }
 
 /* The outermost table container. todo: rename. */
-.app_table_container {
+div.app_table_container {
```

Comment:
I thought this class was specific to our library and only ever applied to a div in the CollapseTableTransform?